core/web/schema/type/api_token.graphql

```diff
@@ -0,0 +1,30 @@
+type APIToken {
+    accessKey: String!
+    secret: String!
+}
+
+input CreateAPITokenInput {
+    password: String!
+}
+
+type CreateAPITokenSuccess {
+    token: APIToken!
+}
+
+union CreateAPITokenPayload = CreateAPITokenSuccess | InputErrors
+
+input DeleteAPITokenInput {
+    password: String!
+}
+
+type DeleteAPITokenResult {
+    accessKey: String!
+    # Not returning the secret since it is hashed in the DB
```

Comment:
Yes that's correct you don't need to return the API token. You can remove this comment